index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Scoped CSS Variables and JS</title>
+  </head>
+  <body>
+    <h2>Update CSS Variables with <span class="hl">JS</span></h2>
+    <div class="container">
+      <div class="controls">
+        <label for="spacing">Spacing:</label>
+        <input id="spacing" type="range" name="spacing" min="10" max="100" value="10" data-sizing="px"/>
+        <label for="blur">Blur:</label>
+        <input id="blur" type="range" name="blur" min="0" max="5" value="0" data-sizing="px"/>
+        <label for="base">Base Color</label>
+        <input id="base" type="color" name="base" value="#ffc600"/>
+        <label for="grayscale">Grayscale:</label>
+        <input id="grayscale" type="range" name="grayscale" min="0" max="100" value="0" data-sizing="%">
+        <label for="invert">Invert:</label>
+        <input id="invert" type="range" name="invert" min="0" max="100" value="0" data-sizing="%">
+        <label for="saturate">Saturate:</label>
+        <input id="saturate" type="range" name="saturate" min="100" max="200" value="100" data-sizing="%">
+        <label for="hue-rotate">Hue-rotate:</label>
+        <input id="hue-rotate" type="range" name="hue-rotate" min="0" max="360" value="0" data-sizing="deg">
+        <label for="sepia">Sepia:</label>
+        <input id="sepia" type="range" name="sepia" min="0" max="100" value="0" data-sizing="%">
+      </div>
+      <img src="photo.jpg" alt="photo"/>
+    </div>
+    <script src="script.js"></script>
+  </body>
+</html>
```

Comment:
Check out the validation errors: https://validator.w3.org/nu/?doc=https%3A%2F%2Fkistanislav.github.io%2FCSSVariables%2Findex.html

Reply:
DONE!